src/Symfony/Component/Notifier/Bridge/FakeSms/FakeSmsTransport.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\FakeSms;
+
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\Mailer\Exception\TransportExceptionInterface;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author James Hemery <james@yieldstudio.fr>
+ */
+final class FakeSmsTransport extends AbstractTransport
```

Comment:
```suggestion
final class FakeSmsMailTransport extends AbstractTransport
```

So that we can add other types like log and database later:
![CleanShot 2021-03-22 at 10 11 29](https://user-images.githubusercontent.com/995707/111966318-13e10000-8af7-11eb-9c15-cf69eb8a224e.png)

![CleanShot 2021-03-22 at 10 11 40](https://user-images.githubusercontent.com/995707/111966329-16435a00-8af7-11eb-8004-2c9a2a2ab5f5.png)